Makefile

```diff
@@ -281,14 +281,24 @@ cmd/migrate/migrate:
 #############
 GOX = gox $(GO_FLAGS) -output="dist/{{.Dir}}-{{.OS}}-{{.Arch}}"
 CGO_GOX = gox $(DYN_GO_FLAGS) -cgo -output="dist/{{.Dir}}-{{.OS}}-{{.Arch}}"
+
+SKIP_ARM ?= false
 dist: clean
+ifeq ($(SKIP_ARM),true)
```

Comment:
(nit) The name SKIP_ARM is a bit misleading. You're swapping arm for Intel here.

Reply:
intel is being built in either case, so the guard is to whether _only_ to build intel, or whether to build intel _and_ arm, unless I'm missing something? problem is GitHub actions don't support arm yet, so we need a way to disable those archs specifically